lib/src/utils.dart

```diff
@@ -17,24 +18,8 @@ String basicAuthHeader(String identifier, String secret) {
 
 void addTrackingHeaders(Map<String, String> headers, String? trackingId, Map<String, String>? additionalHeaders) {
   if (additionalHeaders != null) {
-    if (additionalHeaders[HttpHeadersConsts.userUID] != null) {
-      headers[HttpHeadersConsts.userUID] = additionalHeaders[HttpHeadersConsts.userUID]!;
-    }
-    if (additionalHeaders[HttpHeadersConsts.globalUserId] != null) {
-      headers[HttpHeadersConsts.globalUserId] = additionalHeaders[HttpHeadersConsts.globalUserId]!;
-    }
-    if (additionalHeaders[HttpHeadersConsts.trackingId] != null) {
-      headers[HttpHeadersConsts.trackingId] = additionalHeaders[HttpHeadersConsts.trackingId]!;
-    }
-    if (additionalHeaders[HttpHeadersConsts.applicationVersion] != null) {
-      headers[HttpHeadersConsts.applicationVersion] = additionalHeaders[HttpHeadersConsts.applicationVersion]!;
-    }
-    if (additionalHeaders[HttpHeadersConsts.appVersion] != null) {
-      headers[HttpHeadersConsts.appVersion] = additionalHeaders[HttpHeadersConsts.appVersion]!;
-    }
-    if (additionalHeaders[HttpHeadersConsts.clientOs] != null) {
-      headers[HttpHeadersConsts.clientOs] = additionalHeaders[HttpHeadersConsts.clientOs]!;
-    }
+    headers.addAll(additionalHeaders);
+    headers.remove(HttpHeaders.contentTypeHeader);
```

Comment:
Tohle imo taky nebude dobře 😄 ... Pokud se nepletu je to společná metoda jak při refreshi tak i během normálního requestu že? Při běžném reqeustu contentType určitě musí zůstat... API request třeba potřebuje JSON. 

Nevím jak přesně refrresh OAUTH request má být poskládaný, ale jestli tam něco nefungovalo možná to bralo právě content-type... a nějaké další hlavičky